intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigDropDown.kt

```diff
@@ -120,9 +122,9 @@ class MirrordConfigDropDown : ComboBoxAction() {
     // this function spawns a background task to query the index, not blocking the current thread
     // it maintains a single task to do so, using the `blockQueries` flag
     private fun queryIndexInSmartMode(project: Project, query: (Project) -> Unit) {
+        updateConfigs = false
         object : Task.Backgroundable(project, "mirrord") {
             override fun run(indicator: ProgressIndicator) {
-                updateConfigs = false
```

Comment:
Hey @infiniteregrets, why did you move the unsetting of `updateConfigs` out of the background task?

Reply:
sorry, sounds like a mistake on my end, i pushed the wrong diff. ill fix this